src/gpu/utils.rs

```diff
@@ -21,3 +21,22 @@ pub fn get_devices(platform_name: &str) -> GPUResult<Vec<Device>> {
         None => Err(GPUError {msg: "GPU platform not found!".to_string() })
     }
 }
+
+use std::collections::HashMap;
+lazy_static!{
+    static ref CORE_COUNTS: HashMap<&'static str, usize> = vec![
+        ("GeForce RTX 2080 Ti", 4352),
```

Comment:
can't the cards report their own core count Oo

Reply:
I couldn't get the right numbers for this from clinfo unfortunately.

Reply:
but this also prevents running the kernel on unknown, untested cards which is good i think  :smiley: